docs/sources/enterprise/license-restrictions.md

```diff
@@ -25,6 +25,12 @@ Restrictions are applied separately for viewers and editors.
 
 When the number of maximum active viewers or editors is reached, Grafana displays a warning banner.
 
+### Usage billing
+
+It is possible to turn on usage billing in enterprise licensing to have unlimited number of user limits. When usage billing is on, Grafana will not enforce active user limits and instead will charge for active users above the limit, according to the customer's contract. 
+
+The usage billing must be agreed with Grafana Labs and is only available to customers who have automated license token renewal enabled.
```

Comment:
```suggestion
The usage billing must be agreed upon with Grafana Labs and is only available to customers who have automated license-token renewal enabled.
```

Reply:
Good enough atm, but sounds like pricing speak. Can we simplify this? I need to eat lunch, so brain on empty atm.

Reply:
Curious why the dash for `license-token`?

Reply:
Looks like a compound adjective.

Reply:
My take:

"Usage billing must be agreed upon with Grafana Labs, and it is only available if you have turned on automatic license token renewal."

I say "automatic license token renewal" because it's the name of a specific feature. 

"automatic license token renewal" should link to docs for that feature, but it looks like those docs do not yet exist. Not to pull the string too hard here, but has anyone looked at docs for automatic license token renewal? I don't even see the config option documented. @sakjur this might have been part of #835

Reply:
#30217

Reply:
Thanks, I didn't see that there is an enterprise-specific configuration page. That's a little confusing; perhaps something to consider as we structure docs pages @osg-grafana

Reply:
👆 I agree and I have brought this up with @osg-grafana before as something we should consider. It's hard to say what's the right option here (how do we differentiate two products which are _almost but critically not quite_ the same), but I think we'd need some clearer and standardized way to flag enterprise-only configuration options intermixed with the regular configuration page.

Reply:
Yes agreed. There is currently debate raging on this in a series of workshops. My opinion is that we should have a filter on the page for Grafana "edition" (OSS, Pro, Advanced, Enterprise) and use tags to highlight premium features. Gitlab does this: https://docs.gitlab.com/ee/user/group/saml_sso/

Reply:
I am going to leave this discussion for a planning session outside a PR.

Reply:
fwiw, looks like GitLab uses Nanoc, and Denis (founder of Nanoc) is a friend of mine.